examples/testclient/hello/app.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from . import conn_app
+
+# reuse the configured logger
+logger = logging.getLogger("uvicorn.error")
```

Comment:
What's the reasong for using the `uvicorn.error` logger instead of a specific one for this file/application?

Reply:
This was basically a cheat, the level is set appropriately on this logger so the output from my code appears together with output from uvicorn, and no additional configuration is necessary.

Reply:
Please say if using the `unicorn.error` logger is a deal breaker and I will change the logging.